Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/submission-grid-row/SubmissionGridRow.tsx

```diff
@@ -84,10 +85,9 @@ const SubmissionGridRow = ({ submission }: ISubmissionGridRowProps) => {
             </div>
             <div className={styles.detailsContainer}>
                 <div>
-                    {/* TODO: Fix this URL once https://github.com/SoftUni-Internal/exam-systems-issues/issues/184 is done */}
                     <LinkButton
                       text={problemName}
-                      to={`/contests/${contestId}/${participationType}`}
+                      to={`/contests/${contestId}/${participationType}#${orderBy + 1}`}
```

Comment:
Is this correct? Order by is a number which can be configured in the administration and sometimes administrators put 10, 20, 30 for tasks 1, 2, 3 so it's easier to put a task between 10 and 20 by setting 11 as a value. Doesn't this mean that the url should contain the plain order by value?

Reply:
Their orderBy is normalized and then recalculated by this function in useProblems() hook, so it should be ok: 

const normalizeOrderBy = (problems: IProblemType[]) => {
    problems.sort((x, y) => x.orderBy - y.orderBy);

    return problems.map((p, i) => ({
        ...p,
        orderBy: i + 1,
    }));
};